### ISSUE TITLE
AutoFit row height of merged cells with wrapped text

### ISSUE THREAD

**Issue by zajjoseph** [SheetJS/sheetjs#426]
Hi,

I am trying to write wrapped text into merged cells, but row height is not adjusted automatically.
It is working well when cells are not merged.
Any solution to fix this, please?

Thanks

**Comment by lumio** [comment]
A working example where `textWrap` would be great!
https://codepen.io/lumio/pen/PJNBjv

**Comment by Suncatcher** [comment]
> A working example where textWrap would be great!

But this example involves JQuery, ya?

**Comment by lumio** [comment]
I'm not sure where you see jQuery. You don't need jQuery at all to generate a XLSX file.

**Comment by Suncatcher** [comment]
> I'm not sure where you see jQuery here. You don't need jQuery at all to generate a XLSX file.

Ok, thanks. I see it works.

And what other properties will work in this context, except height (hpt):

   ws[ '!rows' ][ table.querySelectorAll( 'tr' ).length - 1 ] = {
      hpt: 300,
   };

I haven't found proper documentation. Can I set color or bold/font like this?

**Comment by lumio** [comment]
You can find the documentation for this library [here](https://github.com/SheetJS/js-xlsx).
But you are right, the documentation for this library is not the best. I switched to another Excel JS library for that reason.

**Comment by Suncatcher** [comment]
Yeahm thanks, Already learn it. 
However I see, that there is no full-featured text-wrap. In your fiddle wrapping working only after you edit the cell. I wasn't able to achieve the same behavior for my code.
And do you know, if it is possible to make align?

**Comment by lumio** [comment]
@Suncatcher ... this kind of is not the place for this. Better open up a new issue. Also I didn't achieve much styling with js-xlsx even though it might be possible. I'm using a different library now

**Comment by kitzler-walli** [comment]
@lumio  Could you please tell me which library you are using now?

**Comment by SheetJSDev** [comment]
In general, Excel does not automatically adjust the row heights when opening a file that has a merged cell spanning multiple rows.  The library would have to manually calculate this height and assign it to the row.  That calculation involves fonts and styling, which is a part of our [Pro builds](https://sheetjs.com/pro).

From Excel's perspective it makes some sense.  Suppose a merged cell spans 3 rows and should have a height of 120 pixels.  In absence of the merge cell, suppose each row would have a height of 20 pixels.  The gridlines each take 1 px height.  That means 120 - 3 * 20 - 2 * 1= 58 pixels would have to be added.  Where would they be added?
